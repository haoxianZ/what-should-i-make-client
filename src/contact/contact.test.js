import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Contact from './contact'

describe(`contact form component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<Contact />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})