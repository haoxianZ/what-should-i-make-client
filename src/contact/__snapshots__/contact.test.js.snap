// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contact form component renders the complete form 1`] = `
<div>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <form
    className="contactForm"
    onSubmit={[Function]}
  >
    <label
      htmlFor="subject"
    >
      Subject: 
    </label>
    <br />
    <input
      name="subject"
    />
    <br />
    <label
      htmlFor="message"
    >
      Message: 
    </label>
    <br />
    <textarea
      cols="70"
      name="message"
      required={true}
      rows="10"
    />
    <br />
    <label
      htmlFor="from_name"
    >
      Your Email: 
    </label>
    <br />
    <input
      name="from_name"
      type="email"
    />
    <input
      type="submit"
      value="Send"
    />
  </form>
</div>
`;
